shared/src/commonMain/kotlin/com/myprojectname/app/ui/components/PetCardSmall.kt

```diff
@@ -34,19 +35,18 @@ import com.myprojectname.app.ui.theme.Typography
 import io.kamel.image.KamelImage
 import io.kamel.image.asyncPainterResource
 
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun PetCardSmall(
     item: PetModel,
-    width: Dp,
```

Comment:
Why you removed the option of setting a custom size for the Pet Card? This that this component is used in 2 places that need a different size.